### DISCUSSION TITLE
2021 Hacktoberfest Participation

### DISCUSSION THREAD

**Discussion by personalizedrefrigerator** [xournalpp/xournalpp#3380] [Ideas]
## Summary

Hacktoberfest is a program run by Digital Ocean that encourages contributions to open source projects. Maintainer information can be found here: https://hacktoberfest.digitalocean.com/resources/maintainers.

## For us to participate...

 * [**Suggested**] This year, we need to add the `hacktoberfest` label/topic to our repository.
 * [**Suggested**] Add the `hacktoberfest` label to issues new contributors might be interested in.
 * [**Suggested**] Write a guide for new contributors and a code of conduct.
 * See https://hacktoberfest.digitalocean.com/resources/maintainers for additional suggestions.

## Some considerations

 * It looks like Xournal++ has participated in Hacktoberfest in the past. How many pull requests should we expect?
   * It looks like many pull requests have been low-quality or spam for participating repositories in the past, but [Digital Ocean has/is planning to introduce changes to limit this](https://web.archive.org/web/20201210133026/https://hacktoberfest.digitalocean.com/hacktoberfest-update)...
 * If we should expect a large number of pull requests, does the community have time to review and accept (merge the PR or add the `hacktoberfest-accepted` label to it) these contributions?

**Comment by Technius** [comment]
> It looks like Xournal++ has participated in Hacktoberfest in the past. How many pull requests should we expect? 

From my experience, only slightly more activity--not a significant amount.

> does the community have time to review and accept these contributions?

Not sure. Would have to check with the others.
